src/sbroker_util.erl

```diff
@@ -74,7 +74,8 @@ sojourn_target(#{target := _} = Spec) ->
 sojourn_target(#{}) ->
     native(100).
 
--spec sojourn_target(Queue, #{Queue => #{target => Target}}) -> NTarget when
+-spec sojourn_target(Queue, QueueConfig) -> NTarget when
```

Comment:
This, and similar changes, are made to prevent breakage in the building of the doc.

Reply:
Fun stuff!